integration-tests/docker-base/setup.sh

```diff
@@ -34,14 +34,30 @@ apt-get install -y mysql-server
 apt-get install -y supervisor
 
 # Zookeeper
-wget -q -O - http://www.us.apache.org/dist/zookeeper/zookeeper-3.4.13/zookeeper-3.4.13.tar.gz | tar -xzf - -C /usr/local \
-  && cp /usr/local/zookeeper-3.4.13/conf/zoo_sample.cfg /usr/local/zookeeper-3.4.13/conf/zoo.cfg \
-  && ln -s /usr/local/zookeeper-3.4.13 /usr/local/zookeeper
+wget -q -O /tmp/zookeeper-3.4.14.tar.gz "http://www.us.apache.org/dist/zookeeper/zookeeper-3.4.14/zookeeper-3.4.14.tar.gz"
```

Comment:
how about using this one https://apache.org/dist/zookeeper/zookeeper-3.4.14/zookeeper-3.4.14.tar.gz

Reply:
Sure, is it better for some reason?

Reply:
probably because that is "https" , we can probably do same for kafka url below.

Reply:
Ah I didn't notice the https. I guess I can remove the shasum checks too. What do you all think?

Reply:
sure

Reply:
OK, updated.